packages/components/src/notice/stories/index.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import Notice from '..';
+import type { NoticeProps } from '../types';
+
+const meta: ComponentMeta< typeof Notice > = {
+	title: 'Components/Notice',
+	component: Notice,
+	argTypes: {
+		isDismissible: { control: 'boolean' },
+		onDismiss: { control: { type: null } },
+		onRemove: { control: { type: null } },
+		politeness: {
+			control: 'radio',
+			options: [ 'assertive', 'polite' ],
+		},
+		spokenMessage: { control: 'text' },
+		status: {
+			control: 'radio',
+			options: [ 'warning', 'success', 'error', 'info' ],
+		},
+	},
+	parameters: {
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
+	},
+};
+export default meta;
+
+const Template = ( props: NoticeProps ) => {
```

Comment:
Boilerplate-reducing tip: We can annotate the `Template` as a `ComponentStory`, and then the other stories extending it with `.bind()` don't need an explicit type annotation 😎 

```suggestion
const Template: ComponentStory< typeof Notice > = ( props ) => {
```

Reply:
🧠 brilliant, thanks!